### ISSUE TITLE
Unusably small pins on home screen map

### ISSUE THREAD

**Issue by ltseng** [techmatters/terraso-mobile-client#221]
## Description
The pins that currently appear in the latest build of the Android app are too small to be used.  They should be at least as large as described by the Material Design guidelines on [Accessibility](https://m3.material.io/foundations/accessible-design/accessibility-basics) and [Icon Buttons](https://m3.material.io/components/icon-buttons/overview). This is a bug we'll need to fix before getting to usability studies.
![image](https://github.com/techmatters/terraso-mobile-client/assets/2730009/222aba29-6a7a-4b07-bc23-978760c7e818)

## Steps To Reproduce

1. Open up the app
2. Attempt to select any of the existing pins

## Expected behavior

The pins should be easily selectable.

## Actual behavior
The pins are not easily selectable.
